app/helpers/products_helper.rb

```diff
@@ -1,16 +1,10 @@
 module ProductsHelper
 
   def price_policy_errors(product)
-    @facility_price_groups ||= current_facility.price_groups.count
-    price_policy_count = product.price_policies.current.count
-    error_msg = ""
-    if price_policy_count == 0
+    if product.is_a?(Bundle)
```

Comment:
I don't know if it'd be clearer, but the `is_a?` makes me wonder if you might be better off implementing `missing_price_policy?` (or `has_price_policy?`) on `Product` and `Bundle`, though you'd still have to switch on the error message I suppose.